testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js

```diff
@@ -135,6 +135,25 @@ module('buttons group rendering', () => {
         instance.option('dataSource', [1, 2, 3]);
         assert.strictEqual(onContentReadyHandler.callCount, 2);
     });
+
+    test('onContentReady - subscription using "on" method', function(assert) {
+        const done = assert.async(1);
```

Comment:
```suggestion
        const done = assert.async();
```
1 is a default value.